instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -22,6 +22,8 @@
 
 # Imports #####################################################################
 import ddt
+from freezegun import freeze_time
```

Comment:
@jamestait, nit: we group all the `import *` together followed by the `from * import *`.

Reply:
Ah, yes, sorry about this - I'll fix that.

Reply:
This import wasn't necessary (see conversation below), so it will be removed.